src/WinRT.Runtime/ObjectReference.cs

```diff
@@ -36,6 +38,10 @@ public IntPtr ThisPtr
             }
         }
 
+        public bool IsFreeThreaded => _contextPtr == IntPtr.Zero;
```

Comment:
Why does CsWinRT not have almost any XML docs? Should we not have them, at least for public APIs?

Reply:
In theory we should have XML docs and may add them at some point.  But at the same time, most of our public APIs other than the Marshaler APIs are intended for generated projection consumption and not to be called directly by consumers.